### PR TITLE
Support for default array values

### PR DESCRIPTION
**Pull request by pzduniak**
Love the library, unfortunately there's one important missed edge case, namely passing default values with commas in them. This PR fixes it using the simplest possible approach. We're using it in production without any issues.

I won't bother with adding tests before getting an OK on the approach from you.

### CODE REVIEW

File: envconfig.go [hunk 1]
```diff
@@ -324,13 +324,15 @@ func keyAndOpts(tag string) (string, *options, error) {
 	key, tagOpts := strings.TrimSpace(parts[0]), parts[1:]
 
 	var opts options
-	for _, o := range tagOpts {
+optsLoop:
```
Comment (sethvargo) [inline envconfig.go:327]: Style thing: can you use an all uppercase name for the loop please?

**Comment by pzduniak** [comment]
Cool, thanks for the review, I'll do it next week :)

**Comment by sethvargo** [comment]
Bump
